src/screens/EthToSub.tsx

```diff
@@ -19,15 +19,15 @@ import useRialtoBlocks from '../hooks/useRialtoBlocks';
 import shortenAddress from '../util/shortenAddress';
 
 interface Props {
-    className?: string
-    ethProvider: ethers.providers.Web3Provider
+    className?: string;
+	ethProvider: ethers.providers.Web3Provider;
```

Comment:
tabs?

Reply:
yup, interesting that the linting didn't catch that